src/modules/delete-complete-tasks.js

```diff
@@ -0,0 +1,12 @@
+const deleteAllComplete = (task) => {
+  const newArray = [];
+  for (let i = 0; i < task.length; i += 1) {
+    if (task[i].completed === false) {
+      newArray.push(task[i]);
+    }
+  }
+  task.length = 0;
+  task.push(...newArray);
+};
+
+export default deleteAllComplete;
```

Comment:
I think you can make a single file for handling and deleting completed tasks as their functionality is related to each other instead of two of them separately.